CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ From version 2.6.0, the sections in this file adhere to the [keep a changelog](h
 
 ### Fixed
 * [#2030](https://github.com/Shopify/shopify-cli/pull/2030): Fix Theme::Syncer handling of file deletions in `download_file!`
+* [#2070](https://github.com/Shopify/shopify-cli/pull/2070): Improve Nokogiri LoadError message
```

Comment:
This is just a rewording, I don't think it's worth mentioning it here.

```suggestion
```

Reply:
Fair enough. No strong opinions, still finding the right balance!